### PR TITLE
(fix:docs) update static server port to 5042

### PR DESCRIPTION
**Pull request by schalkneethling**
Related to changes in mdn/yari#5134, this updates references to port `5000` to be `5042`

### CODE REVIEW

File: files/es/learn/common_questions/set_up_a_local_testing_server/index.html [hunk 1]
```diff
@@ -2,112 +2,175 @@
 title: ¿Cómo se configura un servidor de prueba local?
```
Comment (SphinxKnight) [inline files/es/learn/common_questions/set_up_a_local_testing_server/index.html:2]: Why is this file changed for this?

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
Several files are transformed in unrelated ways to the PR topic (new lines vs. wrapping)

I'm also not sure if touching Learning Area with other "test server" is necessary.
Edit: for instance https://developer.mozilla.org/es/docs/Learn/Common_questions/set_up_a_local_testing_server is out of date vs en-US but the use of :5000 here is unrelated to yari.

**Comment by schalkneethling** [comment]
> Several files are transformed in unrelated ways to the PR topic (new lines vs. wrapping)

Yeah, that must be my editor. Let me redo some of these. Thank you for the quick feedback @SphinxKnight

**Comment by schalkneethling** [comment]
@SphinxKnight updated.

**Review by SphinxKnight — APPROVED** [review]
Thanks a lot @schalkneethling, I've also added one "contribute" page in French and checked for the rest of the occurrences of "5000" in the whole repo.

Looks good to me.
